docs/scos/dev/feature-integration-guides/202204.0/splittable-order-items-feature-integration.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Splittable Order Items feature integration
+last_updated: Nov 22, 2019
+template: feature-integration-guide-template
+originalLink: https://documentation.spryker.com/v3/docs/splittable-order-items-integration
+originalArticleId: ea499c83-900d-4788-bcaf-c7981220b441
+redirect_from:
+  - /v3/docs/splittable-order-items-integration
+  - /v3/docs/en/splittable-order-items-integration
+---
+
+The Splittable Order Items feature is shipped with following modules:
+
+| Module | Description |
+| --- | --- |
+| [DiscountExtension](https://github.com/spryker/spryker/tree/master/Bundles/DiscountExtension) | Provides extension plugins for the `Discount` module. |
+| [SalesQuantity](https://github.com/spryker/spryker/tree/master/Bundles/SalesQuantity)| Provides support in handling and configuring quantity for sales orders and items. |
+
+To install the Merchants and Merchant relations feature, follows the steps below:
+
+1. Install necessary modules using composer
+Update existing and install the required modules:
+
+```bash
+composer update "spryker/*" "spryker-shop/*"
```

Comment:
I don't think this is really needed
we have to mention exact modules needs to be updated to get new functionality

Reply:
this came from the previous version of the document. Removed.